app/Http/Controllers/Product/ExtendedBaseProductController.php

```diff
@@ -175,10 +175,9 @@ public function adminDownload($id, $invoice = '', $api = false)
      */
     public function saveCartValues($input, bool $can_modify_agent, bool $can_modify_quantity)
     {
-        $this->product->show_agent = $input['show_agent'] == 1 ? 1 : 0; //if Show Agents Selected
+        $this->product->show_agent = $input['show_agent'] == 1 ?: 0; //if Show Agents Selected
```

Comment:
only
```
 $this->product->show_agent = $input['show_agent'] == 1
```
will do the same thing as the entire statement, plus less processing power needed

Reply:
fixed